### DISCUSSION TITLE
How to install an older version of deepmd using conda

### DISCUSSION THREAD

**Discussion by mayankaditya** [deepmodeling/deepmd-kit#408] [Q&A]
Hi,

I have upgraded deepmd-kit using conda. But it's not supporting the previously trained model. So I want to switch back to the previous version 1.2. Please let me know how I can do that.

Thanks,
Mayank

**Comment by njzjz** [comment]
You many use something like `conda install deepmd-kit=1.2.*=*gpu -c deepmodeling`
